calculations.html

```diff
@@ -133,7 +133,6 @@ <h3 class="result-header">Resultat</h3>
 			</div>
 		</body>
 		<script src="js/configVariables.js"></script>
-		<script src="js/APIMiddleware.js"></script>
```

Comment:
??? This will probably break stuff???
What about fetching pumps and validating the users token?